CRM/Report/Form/Mailing/Summary.php

```diff
@@ -350,6 +350,24 @@ public function select() {
       'civicrm_mailing_event_unsubscribe',
     );
 
+    // Define a list of columns that should be counted with the DISTINCT
+    // keyword. For example, civicrm_mailing_event_opened.unique_open_count
+    // should display the number of unique records, whereas something like
+    // civicrm_mailing_event_opened.open_count should display the total number.
+    // Each string here is in the form $tableName.$fieldName, where $tableName
+    // is the key in $this->_columns, and $fieldName is the key in that array's
+    // ['fields'] array.
+    // Reference: CRM-20660
+    $distinct_count_columns = array(
```

Comment:
Can you please change this to satisfy civi-coding standards([camelCase](https://wiki.civicrm.org/confluence/display/CRMDOC/PHP+Code+and+Inline+Documentation)). It would also be nice to fix `$count_tables` above.